models/plant.js

```diff
@@ -11,10 +11,11 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      Plant.hasMany(models.GardenPlant);
+      Plant.belongsToMany(models.Garden, { through: models.GardenPlant });
```

Comment:
Here we see the data model ASSOCIATIONS defined between Garden/GardenPlant/Plant.